code/modules/power/singularity/containment_field.dm

```diff
@@ -66,6 +66,7 @@
 
 /obj/machinery/field
 	var/hasShocked = 0 //Used to add a delay between shocks. In some cases this used to crash servers by spawning hundreds of sparks every second.

```

Comment:
Because it isn't taken into account at all when you throw objects at the field, only mobs